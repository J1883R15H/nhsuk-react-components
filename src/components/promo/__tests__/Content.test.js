import React from 'react';
import { shallow } from 'enzyme';
import Promo from '../promo';

describe('content', () => {
  it('matches snapshot and element', () => {
    const shallowMount = shallow(<Promo.Content>Hello</Promo.Content>);
    expect(
      shallowMount.containsMatchingElement(
        <p className="nhsuk-promo__description">Hello</p>
      )
    ).toBeTruthy();
    expect(shallowMount).toMatchSnapshot();
    shallowMount.unmount();
  });
});
