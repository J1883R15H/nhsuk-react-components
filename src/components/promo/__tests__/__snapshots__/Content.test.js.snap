// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`content matches snapshot and element 1`] = `
<p
  className="nhsuk-promo__description"
>
  Hello
</p>
`;
